app/models/meeting.js

```diff
@@ -75,15 +75,6 @@ export default Model.extend({
     return this.isFinal && !this.releasedDocuments;
   }),
 
-  latestAgenda: computed('agendas.[]', function() {
-    return PromiseObject.create({
-      promise: this.get('agendas').then((agendas) => {
-        const sortedAgendas = agendas.sortBy('agendaName').reverse();
-        return sortedAgendas.get('firstObject');
-      }),
-    });
-  }),
-
```

Comment:
I think you removed `latestAgenda` because of my review?
However, I said `latestAgendaǸame` can go.
`latestAgenda` computed still has more usages (agenda-activity model, pods/agenda/overview, pods/newsletter)

Reply:
refactor of those 3 places is out of scope.

Reply:
Potverdorie, I blame lack of coffee for this... Thanks for the catch Sven!